Sources/MapboxMaps/Offline/OfflineManager+MapboxMaps.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+
+extension MapboxCoreMaps.OfflineManager {
+    /// Loads a new style package or updates the existing one.
+    ///
+    /// - Parameters:
+    ///   - styleURI: The URI of the style package's associated style
+    ///   - loadOptions: The style package load options.
+    ///   - progress: Invoked multiple times to report progress of the loading
+    ///         operation.
+    ///   - completion: Invoked only once upon success, failure, or cancelation
+    ///         of the loading operation. Any `Result` error should be of type
+    ///         `StylePackError`.
```

Comment:
Strictly speaking, it could also be of type `OfflineError`, right? I don't think we should make any claims here so that we're free to change the implementation later.